src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -346,6 +347,10 @@ Hides the copyright banner at startup.
 
     Example: "pwsh -o XML -c Get-Date"
 
+-OutputLog | -ol
```

Comment:
What about using ConPTY/PTY?

Reply:
You mean with pwshw?  There is no PTY attached since it's not a console app.

Reply:
Yes. I see you have to add many workarounds in console host code.  I guess using PTY simplify our pwshw related code.